README.md

```diff
@@ -452,6 +491,18 @@ The `session` argument should be a session if found, otherwise `null` or
 `undefined` if the session was not found (and there was no error). A special
 case is made when `error.code === 'ENOENT'` to act like `callback(null, null)`.
 
+### store.load(sid, callback)
+
+**Required** (available from `express-session`)
+
+This required method is used to load a `Session` instance via the given session ID (`sid`). The `callback` should be called as `callback(error, session)`. This method calls `store.createSession` internally.
+
+### store.regenerate(req, callback)
+
+**Required** (available from `express-session`)
+
+This required method is used to regenerate the given request's (`req`) session. The `callback` should be called as `callback(error)`.
```

Comment:
Can you describe in more detail what "regenerate" means? How would I go about writing this method if I'm not using the one from the module?